packages/nibijs/src/test/faucet.test.ts

```diff
@@ -14,7 +14,8 @@ import {
 } from "../tx"
 import { TEST_CHAIN, TEST_MNEMONIC } from "./helpers"
 
-test("faucet utility works", async () => {
+// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Are we sure we want to skip this test?

Reply:
looks like it's gonna be the case, we can't create a test token even with [faked tokens](https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do), this not only would require to create a setup to generate a token from the UI, but to also create a fake backend recaptcha check with a test secret. In this case, we can skip the test and check it manually

Reply:
Can you add this description as a comment in the code?